src/client/client-controller.ts

```diff
@@ -1,42 +1,55 @@
 import { Game, GameState, InputFrame } from '../core/game';
 import { Event } from '../core/event';
 
-// TODO: Move to a core file for interop
-const UPDATE_EVENT = 'state update';
-const SEND_EVENT = 'new frames';
+import { SEND_STATE_UPDATE, StatePayload } from '../server/server-interface';
+import { SEND_FRAMES } from './client-interface';
 
 export class ClientController {
   game: GameState;
   socket: SocketIOClient.Socket;
-  serverUpdate: GameState | null;
+  serverUpdate: StatePayload | null = null;
```

Comment:
lol k thats confusing af syntax but i guess that's what happens when type comes after var name